docs/kola.md

```diff
@@ -236,3 +236,10 @@ In order to see the logs for these tests you must enter the `tmp/kola/name_of_th
 
 `cosa run -i ignition_path` You can run it passing your Ignition, or the Ignition used in the the test that failed for troubleshooting reasons.
 
+## Run test on cloud
+`cosa kola run -p aws --aws-type t3.micro --aws-ami ami-0431766f2498820b8 --aws-region us-east-1 basic` This will run the basic tests on AWS using `ami-0431766f2498820b8` (fedora-coreos-37.20230227.20.2), how to create the credentials refer to https://github.com/coreos/coreos-assembler/blob/main/docs/mantle/credentials.md#aws
```

Comment:
you can leave `--aws-type t3.micro` out and the test framework will use the defaults. I don't even know if t3.micro meets the minimum requirements for FCOS/RHCOS anyway.